content/tutorials/publish-subscribe.textile

```diff
@@ -745,16 +745,18 @@ h2(#live-demo). Live demo
   <div style="margin-top: 5px">Scan this on your mobile</div>
 </div>
 
-<a href="#" id="new-browser" target="_blank">Open this demo in a new browser window</a> to see publish & subscribe in action.  Alternatively, scan the QR code to see the published message delivered in real time.
+<a href="#" id="new-browser" target="_blank">Open this demo in a new browser window</a> to see publish & subscribe in action.  
```

Comment:
> to see publish & subscribe

Can we use 'and' instead of '&' in this case?

Reply:
Good shout! Done.